README.md

```diff
@@ -278,6 +279,7 @@ A curated list of awesome research papers, datasets and software projects devote
 * [gemini](https://github.com/src-d/gemini) - Source code deduplication as scale, production.
 * [enry](https://github.com/src-d/enry) - Insanely fast file based programming language detector.
 * [hercules](https://github.com/src-d/hercules) - Git repository mining framework with batteries on top of go-git.
+* [DeepCS (Deep Code Search)](https://github.com/guxd/deep-code-search) - Keras and Pytorch implementations of DeepCS (Deep Code Search)
```

Comment:
In the beginning `DeepCS (Deep Code Search)` -> `DeepCS` (no need to duplicate the name many times - `of DeepCS (Deep Code Search)` should be enough in description).
Please add dot in the end.